Modify cost aggregation logic

- Adjust the monthly aggregation values slightly to match the values seen in the AWS Management Console.

`isSameOrAfter` and `isSameOrBefore` is not available from generated `index.js`

```bash
❯ pnpm exec node dist/index.js
node:internal/modules/run_main:115
    triggerUncaughtException(
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/---/dev/aws-cost-cli/node_modules/dayjs/plugin/isSameOrAfter' imported from /Users/---/dev/aws-cost-cli/dist/index.js
Did you mean to import "dayjs/plugin/isSameOrAfter.js"?
```

--- a/.github/workflows/check.yaml
+++ b/.github/workflows/check.yaml
@@ -21,11 +21,14 @@ jobs:
           node-version-file: '.node-version'
           cache: 'pnpm'
 
-      - name: Install dependencies
-        run: make install
+      - name: Install dependencies & build application
+        run: make
 
       - name: Lint & format
         run: make lint
 
       - name: Run tests
         run: make test
+
+      - name: Confirm no error
+        run: make version

--- a/Makefile
+++ b/Makefile
@@ -29,3 +29,6 @@ lint:
 
 format:
 	pnpm format
+
+version:
+	pnpm exec node $(BUILD_DIR)/index.js --version

--- a/src/cost.ts
+++ b/src/cost.ts
@@ -1,7 +1,7 @@
 import AWS from 'aws-sdk';
 import dayjs from 'dayjs';
-import isSameOrAfter from 'dayjs/plugin/isSameOrAfter';
-import isSameOrBefore from 'dayjs/plugin/isSameOrBefore';
+import isSameOrAfter from 'dayjs/plugin/isSameOrAfter.js';
+import isSameOrBefore from 'dayjs/plugin/isSameOrBefore.js';
 dayjs.extend(isSameOrAfter);
 dayjs.extend(isSameOrBefore);
 import { AWSConfig } from './config';

